src/audiences/ui/components/rule.js

```diff
@@ -64,14 +64,24 @@ const RuleInput = props => {
 	} = props;
 
 	// Get the currently available values for the field.
-	const data = ( currentField.data && currentField.data.map( datum => datum.value ) ) || [];
+	const options = ( currentField.data && currentField.data.map( option => ( {
+		value: option.value,
+		label: ( option?.label ? `${ option.label } (${ option.value })` : option.value ) + String( option?.percent ? ` ~${ option.percent }%` : '' ),
```

Comment:
`option?.` is checking that `option` isn't `null` so it seems redundant for these checks. It'd make sense if we did the same for `option.value` above but at that point you could run a filter before the map on the `data` array to remove any undefined or falsey entries.

Reply:
Hmmm, I think `option?.label` checks if `option` has a `label` value, rather than it being null. As not all values have a `label` but will always have a `value`.

Reply:
.. or have I been using it wrong all this time ? :D

Reply:
Yeah, seems like I was wrong, this is only useful in chained calls, `option.label` will return undefined anyway if it doesn't exist.

ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
```suggestion
		label: ( option.label ? `${ option.label } (${ option.value })` : option.value ) + String( option.percent ? ` ~${ option.percent }%` : '' ),
```